labgrid/driver/sshdriver.py

```diff
@@ -101,9 +101,9 @@ def _check_master(self):
     @Driver.check_active
     @step(args=['cmd'], result=True)
     def run(self, cmd, codec="utf-8", decodeerrors="strict", timeout=None): # pylint: disable=unused-argument
-        return self._run(cmd, codec=codec, decodererrors=decodeerrors)
+        return self._run(cmd, codec=codec, decodeerrors=decodeerrors)
 
-    def _run(self, cmd, codec, decodeerrors): # pylint: disable=unused-argument
+    def _run(self, cmd, codec="utf-8", decodeerrors="strict", timeout=None): # pylint: disable=unused-argument
```

Comment:
If timeout value is passed on from `run()` to `_run()`, I don't see why any default values are needed here.